doc/wiki/User-Guide/Projects/Project.md

```diff
@@ -222,6 +224,22 @@ The history contains all activities done directly refer to the current project.
 
 ![show project history](../../uploads/Screenshots/view_project_history.jpg)
 
+## Filter the history of a project
+
+**Description:**
+
+The entries in the history can be filtered by 4 different values: publisher, type of event, a start date and a end date, where a timeframe can be determined. When clicking o the search button, all values that are specified are used in the filter request.
```

Comment:
When clicking o the search button...
Suggestion:
By clicking the search button...

Reply:
done

Reply:
2 - done